packages/cicero-core/test/clause.js

```diff
@@ -601,6 +573,16 @@ Assignment. Licensee may freely assign or otherwise transfer all or any of its r
             newText.should.eql(text);
         });
 
+        it('should be able to draft a copyright license (markdown_cicero)', async function() {
+            const template = await Template.fromDirectory('./test/data/copyright-license', options);
+            const clause = new Clause(template);
+            const data = copyrightData;
+            clause.setData(data);
+            const newText = clause.draft({format:'markdown_cicero'});
+            // remove the generated clause id
```

Comment:
I don't understand this comment "`remove the generated clause id`"

Reply:
yes it makes no sense. cut-paste from previous test which had a non-sensical comment!

Reply:
fixed!

Reply:
fixed now...